gc/stats/LargeObjectAllocateStats.cpp

```diff
@@ -133,7 +133,7 @@ MM_LargeObjectAllocateStats::initialize(MM_EnvironmentBase *env, uint16_t maxAll
 
 	for (uintptr_t sizeClassIndex = 0; sizeClassIndex < _freeEntrySizeClassStats._maxSizeClasses; sizeClassIndex++) {
 		/* TODO: this is rather an approximation (at least due to insufficient precision of double math) */
```

Comment:
I'm not sure what's the meaning of this TODO.  We can remove the whole comment.

Reply:
Could you dump the content of _sizeClassSizes array and see if there is a difference when using pow and powf. Although even if there is a difference, I don't see a problem - it's just me being curious.

Reply:
@amicic There is no difference for _sizeClassSizes array(Integer) between pow and powf

sizeClassIndex(0), _sizeClassRatio(1.200000), powf(1), pow(1) 
sizeClassIndex(1), _sizeClassRatio(1.200000), powf(1), pow(1) 
sizeClassIndex(2), _sizeClassRatio(1.200000), powf(1), pow(1) 
sizeClassIndex(3), _sizeClassRatio(1.200000), powf(1), pow(1) 
sizeClassIndex(4), _sizeClassRatio(1.200000), powf(2), pow(2) 
sizeClassIndex(5), _sizeClassRatio(1.200000), powf(2), pow(2) 
sizeClassIndex(6), _sizeClassRatio(1.200000), powf(2), pow(2) 
sizeClassIndex(7), _sizeClassRatio(1.200000), powf(3), pow(3) 
sizeClassIndex(8), _sizeClassRatio(1.200000), powf(4), pow(4) 
sizeClassIndex(9), _sizeClassRatio(1.200000), powf(5), pow(5) 
sizeClassIndex(10), _sizeClassRatio(1.200000), powf(6), pow(6) 
sizeClassIndex(11), _sizeClassRatio(1.200000), powf(7), pow(7) 
sizeClassIndex(12), _sizeClassRatio(1.200000), powf(8), pow(8) 
sizeClassIndex(13), _sizeClassRatio(1.200000), powf(10), pow(10) 
sizeClassIndex(14), _sizeClassRatio(1.200000), powf(12), pow(12) 
sizeClassIndex(15), _sizeClassRatio(1.200000), powf(15), pow(15) 
sizeClassIndex(16), _sizeClassRatio(1.200000), powf(18), pow(18) 
sizeClassIndex(17), _sizeClassRatio(1.200000), powf(22), pow(22) 
sizeClassIndex(18), _sizeClassRatio(1.200000), powf(26), pow(26) 
sizeClassIndex(19), _sizeClassRatio(1.200000), powf(31), pow(31) 
sizeClassIndex(20), _sizeClassRatio(1.200000), powf(38), pow(38) 
sizeClassIndex(21), _sizeClassRatio(1.200000), powf(46), pow(46) 
sizeClassIndex(22), _sizeClassRatio(1.200000), powf(55), pow(55) 
sizeClassIndex(23), _sizeClassRatio(1.200000), powf(66), pow(66) 
sizeClassIndex(24), _sizeClassRatio(1.200000), powf(79), pow(79) 
sizeClassIndex(25), _sizeClassRatio(1.200000), powf(95), pow(95) 
sizeClassIndex(26), _sizeClassRatio(1.200000), powf(114), pow(114) 
sizeClassIndex(27), _sizeClassRatio(1.200000), powf(137), pow(137) 
sizeClassIndex(28), _sizeClassRatio(1.200000), powf(164), pow(164) 
sizeClassIndex(29), _sizeClassRatio(1.200000), powf(197), pow(197) 
sizeClassIndex(30), _sizeClassRatio(1.200000), powf(237), pow(237) 
sizeClassIndex(31), _sizeClassRatio(1.200000), powf(284), pow(284) 
sizeClassIndex(32), _sizeClassRatio(1.200000), powf(341), pow(341) 
sizeClassIndex(33), _sizeClassRatio(1.200000), powf(410), pow(410) 
sizeClassIndex(34), _sizeClassRatio(1.200000), powf(492), pow(492) 
sizeClassIndex(35), _sizeClassRatio(1.200000), powf(590), pow(590) 
sizeClassIndex(36), _sizeClassRatio(1.200000), powf(708), pow(708) 
sizeClassIndex(37), _sizeClassRatio(1.200000), powf(850), pow(850) 
sizeClassIndex(38), _sizeClassRatio(1.200000), powf(1020), pow(1020) 
sizeClassIndex(39), _sizeClassRatio(1.200000), powf(1224), pow(1224) 
sizeClassIndex(40), _sizeClassRatio(1.200000), powf(1469), pow(1469) 
sizeClassIndex(41), _sizeClassRatio(1.200000), powf(1763), pow(1763) 
sizeClassIndex(42), _sizeClassRatio(1.200000), powf(2116), pow(2116) 
sizeClassIndex(43), _sizeClassRatio(1.200000), powf(2539), pow(2539) 
sizeClassIndex(44), _sizeClassRatio(1.200000), powf(3047), pow(3047) 
sizeClassIndex(45), _sizeClassRatio(1.200000), powf(3657), pow(3657) 
sizeClassIndex(46), _sizeClassRatio(1.200000), powf(4388), pow(4388) 
sizeClassIndex(47), _sizeClassRatio(1.200000), powf(5266), pow(5266) 
sizeClassIndex(48), _sizeClassRatio(1.200000), powf(6319), pow(6319) 
sizeClassIndex(49), _sizeClassRatio(1.200000), powf(7583), pow(7583) 
sizeClassIndex(50), _sizeClassRatio(1.200000), powf(9100), pow(9100) 
sizeClassIndex(51), _sizeClassRatio(1.200000), powf(10920), pow(10920) 
sizeClassIndex(52), _sizeClassRatio(1.200000), powf(13104), pow(13104) 
sizeClassIndex(53), _sizeClassRatio(1.200000), powf(15725), pow(15725) 
sizeClassIndex(54), _sizeClassRatio(1.200000), powf(18870), pow(18870) 
sizeClassIndex(55), _sizeClassRatio(1.200000), powf(22644), pow(22644) 
sizeClassIndex(56), _sizeClassRatio(1.200000), powf(27173), pow(27173) 
sizeClassIndex(57), _sizeClassRatio(1.200000), powf(32608), pow(32608) 
sizeClassIndex(58), _sizeClassRatio(1.200000), powf(39130), pow(39130) 
sizeClassIndex(59), _sizeClassRatio(1.200000), powf(46956), pow(46956) 
sizeClassIndex(60), _sizeClassRatio(1.200000), powf(56347), pow(56347) 
sizeClassIndex(61), _sizeClassRatio(1.200000), powf(67617), pow(67617) 
sizeClassIndex(62), _sizeClassRatio(1.200000), powf(81140), pow(81140) 
sizeClassIndex(63), _sizeClassRatio(1.200000), powf(97368), pow(97368) 
sizeClassIndex(64), _sizeClassRatio(1.200000), powf(116842), pow(116842) 
sizeClassIndex(65), _sizeClassRatio(1.200000), powf(140210), pow(140210) 
sizeClassIndex(66), _sizeClassRatio(1.200000), powf(168253), pow(168253) 
sizeClassIndex(67), _sizeClassRatio(1.200000), powf(201903), pow(201903) 
sizeClassIndex(68), _sizeClassRatio(1.200000), powf(242284), pow(242284) 
sizeClassIndex(69), _sizeClassRatio(1.200000), powf(290741), pow(290741) 
sizeClassIndex(70), _sizeClassRatio(1.200000), powf(348889), pow(348889) 
sizeClassIndex(71), _sizeClassRatio(1.200000), powf(418667), pow(418667) 
sizeClassIndex(72), _sizeClassRatio(1.200000), powf(502401), pow(502401) 
sizeClassIndex(73), _sizeClassRatio(1.200000), powf(602881), pow(602881) 
sizeClassIndex(74), _sizeClassRatio(1.200000), powf(723458), pow(723458) 
sizeClassIndex(75), _sizeClassRatio(1.200000), powf(868149), pow(868149) 
sizeClassIndex(76), _sizeClassRatio(1.200000), powf(1041779), pow(1041779) 
sizeClassIndex(77), _sizeClassRatio(1.200000), powf(1250135), pow(1250135) 
sizeClassIndex(78), _sizeClassRatio(1.200000), powf(1500162), pow(1500162) 
sizeClassIndex(79), _sizeClassRatio(1.200000), powf(1800195), pow(1800195) 
sizeClassIndex(80), _sizeClassRatio(1.200000), powf(2160234), pow(2160234) 
sizeClassIndex(81), _sizeClassRatio(1.200000), powf(2592281), pow(2592281) 
sizeClassIndex(82), _sizeClassRatio(1.200000), powf(3110737), pow(3110737) 
sizeClassIndex(83), _sizeClassRatio(1.200000), powf(3732885), pow(3732885) 
sizeClassIndex(84), _sizeClassRatio(1.200000), powf(4479462), pow(4479462) 
sizeClassIndex(85), _sizeClassRatio(1.200000), powf(5375355), pow(5375355) 
sizeClassIndex(86), _sizeClassRatio(1.200000), powf(6450426), pow(6450426) 
sizeClassIndex(87), _sizeClassRatio(1.200000), powf(7740512), pow(7740512) 
sizeClassIndex(88), _sizeClassRatio(1.200000), powf(9288615), pow(9288615) 
sizeClassIndex(89), _sizeClassRatio(1.200000), powf(11146338), pow(11146338) 
sizeClassIndex(90), _sizeClassRatio(1.200000), powf(13375606), pow(13375606) 
sizeClassIndex(91), _sizeClassRatio(1.200000), powf(16050728), pow(16050728) 
sizeClassIndex(92), _sizeClassRatio(1.200000), powf(19260874), pow(19260874) 
sizeClassIndex(93), _sizeClassRatio(1.200000), powf(23113050), pow(23113050) 
sizeClassIndex(94), _sizeClassRatio(1.200000), powf(27735662), pow(27735662) 
sizeClassIndex(95), _sizeClassRatio(1.200000), powf(33282794), pow(33282794) 
sizeClassIndex(96), _sizeClassRatio(1.200000), powf(39939356), pow(39939356) 
sizeClassIndex(97), _sizeClassRatio(1.200000), powf(47927228), pow(47927228) 
sizeClassIndex(98), _sizeClassRatio(1.200000), powf(57512676), pow(57512676) 
sizeClassIndex(99), _sizeClassRatio(1.200000), powf(69015216), pow(69015216) 
sizeClassIndex(100), _sizeClassRatio(1.200000), powf(82818256), pow(82818256) 
sizeClassIndex(101), _sizeClassRatio(1.200000), powf(99381912), pow(99381912) 
sizeClassIndex(102), _sizeClassRatio(1.200000), powf(119258296), pow(119258296) 
sizeClassIndex(103), _sizeClassRatio(1.200000), powf(143109968), pow(143109968) 
sizeClassIndex(104), _sizeClassRatio(1.200000), powf(171731968), pow(171731968) 
sizeClassIndex(105), _sizeClassRatio(1.200000), powf(206078368), pow(206078368) 
sizeClassIndex(106), _sizeClassRatio(1.200000), powf(247294048), pow(247294048) 
sizeClassIndex(107), _sizeClassRatio(1.200000), powf(296752864), pow(296752864) 
sizeClassIndex(108), _sizeClassRatio(1.200000), powf(356103456), pow(356103456) 
sizeClassIndex(109), _sizeClassRatio(1.200000), powf(427324160), pow(427324160) 
sizeClassIndex(110), _sizeClassRatio(1.200000), powf(512788992), pow(512788992)